sdk/extensions/wcf/ci.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```

Comment:
I believe these need to be in the root `/sdk/extensions` folder.   The convention in the repository is to have a default `ci.yml` file that implies "client" and then add the specialized versions as part of the filename.   In this case `ci.wcf.yml`  

Example:  [Event Hubs](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/eventhub) has types for "client', "data", "functions", and "mgmt".